src/components/Timer/Timer.jsx

```diff
@@ -0,0 +1,92 @@
+import "./Timer.css";
+import React, { useRef, useState } from "react";
+import { Duration } from "./Duration";
+import { InputField } from "../InputField";
+import { TimeEntry } from "./TimeEntry";
+
+export const Timer = () => {
+  //state, what I can set or write
+  //getter a setter
+  // setter set it
+
+  //actual time
+  const [nowTime, setNowTime] = useState(null);
+  // start the timer
+  const [startTime, setStartTime] = useState(null);
+
+  const actualTimer = nowTime - startTime;
+
+  const [listTime, setListTime] = useState([]);
+  console.log(listTime);
+
+  // reference of interal - constant
+  const intervalRef = useRef(null);
+
+  const handleStart = () => {
+    //chci aktulni cas, kdy zacla
+    setStartTime(Date.now());
+
+    setNowTime(Date.now());
+    //prop useRef je current
+    intervalRef.current = setInterval(() => {
+      setNowTime(Date.now());
+    }, 1000);
+    //1 sekunda
+  };
+
+  const handleStop = () => {
+    clearInterval(intervalRef.current);
+    console.log(startTime);
+
+    const finalTimePassed = nowTime - startTime;
+
+    //jak dloho ten casovy zaznam trva
+    // const newListTime = [...listTime, finalTimePassed];
+
+    //casovy zaznam co jsem ukoncila a ukladam do sveho stavu - kdyz dam stop
+    const newFinalTimePassed = {
+      id: startTime.toString(),
```

Comment:
FYI: this will work for you, but it is in generally better to let backend to take care of the `id`. There is no backed for now, so you could ignore it.

If, for any reason, the id must been generated on frontend, it is better to use libraries for generating unique IDs (like https://www.npmjs.com/package/nanoid).